<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 3.2.2 (Python 3.9.0 on win32)" generated="20210224 16:31:12.760" rpa="false">
<suite id="s1" name="Landing" source="C:\Users\rjrodriguez\Documents\Rokub0\Clarovideo_Roku\RobotLibrary\Tests\Landing\landing.robot">
<test id="s1-t1" name="01_0001_Verify_channel_is_launched">
<kw name="Launch channel" library="keywords" type="setup">
<kw name="Side load">
<arguments>
<arg>${sideload}</arg>
<arg>${roku_user}</arg>
<arg>${roku_pass}</arg>
</arguments>
<msg timestamp="20210224 16:31:13.278" level="FAIL">No keyword with name 'Side load' found.</msg>
<status status="FAIL" starttime="20210224 16:31:13.278" endtime="20210224 16:31:13.278"></status>
</kw>
<status status="FAIL" starttime="20210224 16:31:13.273" endtime="20210224 16:31:13.279"></status>
</kw>
<status status="FAIL" starttime="20210224 16:31:13.272" endtime="20210224 16:31:13.280" critical="yes">Setup failed:
No keyword with name 'Side load' found.</status>
</test>
<test id="s1-t2" name="Landing's title show correct text">
<kw name="Launch channel" library="keywords" type="setup">
<kw name="Side load">
<arguments>
<arg>${sideload}</arg>
<arg>${roku_user}</arg>
<arg>${roku_pass}</arg>
</arguments>
<msg timestamp="20210224 16:31:13.291" level="FAIL">No keyword with name 'Side load' found.</msg>
<status status="FAIL" starttime="20210224 16:31:13.290" endtime="20210224 16:31:13.291"></status>
</kw>
<status status="FAIL" starttime="20210224 16:31:13.284" endtime="20210224 16:31:13.292"></status>
</kw>
<status status="FAIL" starttime="20210224 16:31:13.282" endtime="20210224 16:31:13.293" critical="yes">Setup failed:
No keyword with name 'Side load' found.</status>
</test>
<doc>Suite Landing</doc>
<status status="FAIL" starttime="20210224 16:31:12.765" endtime="20210224 16:31:13.296"></status>
</suite>
<statistics>
<total>
<stat pass="0" fail="2">Critical Tests</stat>
<stat pass="0" fail="2">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat pass="0" fail="2" id="s1" name="Landing">Landing</stat>
</suite>
</statistics>
<errors>
<msg timestamp="20210224 16:31:13.233" level="ERROR">Error in file 'C:\Users\rjrodriguez\Documents\Rokub0\Clarovideo_Roku\RobotLibrary\Tests\Landing\landing.robot' on line 4: Initializing test library 'RobotLibrary' with arguments [ 192.168.0.200 | 20000 | 2000 | G:/GlobalHitss/Roku/QA/rokub0-qa/src/main.exe ] failed: FileNotFoundError: [WinError 2] El sistema no puede encontrar el archivo especificado
Traceback (most recent call last):
  File "C:\Users\rjrodriguez\Documents\Rokub0\Clarovideo_Roku\RobotLibrary\Library\RobotLibrary.py", line 36, in __init__
    self._process = subprocess.Popen(path)
  File "c:\users\rjrodriguez\appdata\local\programs\python\python39\lib\subprocess.py", line 947, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "c:\users\rjrodriguez\appdata\local\programs\python\python39\lib\subprocess.py", line 1416, in _execute_child
    hp, ht, pid, tid = _winapi.CreateProcess(executable, args,</msg>
</errors>
</robot>
